app/templates/layouts/_base.html

```diff
@@ -49,7 +49,7 @@
     <!-- Google Tag Manager -->
     {% if analytics_gtm_id and analytics_gtm_env_id %}
       <script>
-        dataLayer = [];
+        dataLayer = [{'gtm.blacklist': ['customScripts' , 'html']}];
```

Comment:
What is the reason for blocking these specific tags? Is there any significance to the customScripts and html tags within tag manager?